db/models.js

```diff
@@ -1,27 +1,64 @@
-const  db  = require ("./connection")
+const db = require("./connection")
 
 
 const fetchCategories = () => {
     let sqlString = `SELECT * FROM categories`
     return db.query(sqlString)
-    .then (({rows}) => rows)
+        .then(({ rows }) => rows)
 }
 
 
 const fetchReviews = () => {
-    
-    let sqlString = `SELECT reviews.*, count(*) AS comment_count 
+
+    let sqlString = `SELECT reviews.*, count(comments.review_id) AS comment_count 
```

Comment:
Just be aware that your comment_count is coming back as a string. This is something we will have to accommodate for in FE.